ts/private/tsc.js

```diff
@@ -0,0 +1,3 @@
+const runfiles = require(process.env['BAZEL_NODE_RUNFILES_HELPER'])
+const tsPath = runfiles.resolve('npm_typescript-4.3.5')
```

Comment:
I might have missed something about this feature elsewhere, but where is the 4.3.5 version defined? it looks like @bazel/typescript has a peer dep of `>=3.0.0 <4.4.0`

Reply:
it's in the new `ts/repositories.bzl`.
I fixed this dangling reference, now all the version info is contained within that file.